src/Microsoft.AspNetCore.Razor.VSCode/src/Trace.ts

```diff
@@ -3,12 +3,8 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
-import * as assert from 'assert';
```

Comment:
Messed up diff. Just added a new Trace type that looks like the one from jsonrpc. We need this Trace type in order to write tests. Basically a lot of the VSCode APIs that are shipped don't have implementations outside of VSCode. Therefore, we need to build adapters that look like those types so we can properly test them.